### DISCUSSION TITLE
Beaten Up?

### DISCUSSION THREAD

**Discussion by NoctiBeard** [loathers/garbage-collector#1715] [General]
These last 7 days Ive been randomly beaten up by june cleaver but it doesnt remove the effect anymore. It stops everytime now even though I have tongue of walrus.

**Comment by DEEX3545** [comment]
Did you ever figure this out? I've had the same issue and I've reinstalled everything at this point.

**Comment by spaghetti-squash** [comment]
This seems like an issue caused by running turns outside mafia

**Reply by DEEX3545** [reply]
Is there a method or variable I can reset to sync turns with mafia again? You're right that I sometimes run turns while not logged into mafia.

**Reply by dsimich** [reply]
Adding my 2 cents here, I also encounter this but do not run turns outside of Mafia. I'm assuming it's a me problem vs a garbo problem but I don't know what the solution would be.

**Reply by NoctiBeard** [reply]
I do not run turns outside of mafia or garbo. It seems to be with June cleaver and it stops. However since crimbo started I have not ascended and haven't seen it since. I assume when I ascend again I'll start seeing this.

**Reply by ianknowles** [reply]
Nope, never run turns outside of mafia and its happening more frequently lately. It may be a problem in mafias auto-recovery.

**Comment by gausie** [comment]
What is your `recoveryScript` pref set to?

**Reply by DEEX3545** [reply]
Universal_recovery.ash. If you want the version of it I can grab it if you let me know how.

**Comment by gausie** [comment]
No that’s your problem. Set it blank and try again

**Reply by DEEX3545** [reply]
Clearing my recovery script didn't help unfortuantely.
